rpcd/playbooks/roles/elasticsearch/tasks/upgrade_elasticsearch_pre.yml

```diff
@@ -231,7 +231,7 @@
     - elasticsearch-reindex
 
 - name: Monitor reindexing process
-  shell: /opt/reindex-liberty.py --host "{{ hostvars[groups['elasticsearch_container'][0]]['container_address'] }}" --monitor --continuous
+  shell: /opt/reindex-liberty.py --host "{{ hostvars[groups['elasticsearch_container'][0]]['container_address'] }}" --monitor --continuous --verbose
```

Comment:
I don't see a reason in the commit message for this change (adding `--verbose`).  If this is actually needed, please note why in the commit message.

Reply:
Also, side related, I want to say we needed to tweak --host here to or something. @Frank-ZhangXin mentuioned it in the comments of that card as well.

Reply:
@prometheanfire
"--verbose" is not strictly required here since ansible won't print out monitoring progress in the play, so it can be removed.
@claco
"--host" arg is required if any es operation is asked to be performed.